CONTRIBUTING.md

```diff
@@ -171,6 +172,26 @@ The first line of the comment must be `/black` so that it will be interpreted by
 - [Auto] After the formatting commit, the GitHub action adds an emoji to the comment that triggered the process.
 - Repeat the above steps if necessary.
 
+#### Involve new package
+Except for `torch` and `numpy`, all the other packages are `optional import` for MONAI. All the existing optional packages are listed in [install dependencies](https://docs.monai.io/en/stable/installation.html#installing-the-recommended-dependencies).
+
+Sample code of import logic:
+```py
+from monai.utils import min_version, optional_import
+
+IgniteEngine, _ = optional_import("ignite.engine", 0.4.8, min_version, "Engine")
+
+
+class Workflow(IgniteEngine):
+    ...
+```
+If any of the newly added test program depends on the optional packages, please add it to the `exclude cases` of [min_tests](https://github.com/Project-MONAI/MONAI/blob/dev/tests/min_tests.py) to skip the minimal testing for it.
+
+To involve and config a new optional package in a pull request, please add the necessary information to below files:
```

Comment:
"To include and configure... " You might want to also explain very briefly what every file is meant for.